ar_SA.json

```diff
@@ -244,6 +244,22 @@
 	"certificate.deploy.inprogress.modal.title": "Deploying Certificate",
 	"certificate.deploy.success.modal.message": "The certificate has been successfully deployed.",
 	"certificate.deploy.success.modal.title": "Certificate Deployed",
+	"certificate.dns01chanllenge.error.another.instance": "अस्माकं प्रणाली अन्यस्य अनुरोधस्य प्रमाणीकरणे व्यस्ता अस्ति। कृपया कतिपयेषु निमेषेषु पुनः प्रयासं कुर्वन्तु।",
+	"certificate.dns01chanllenge.error.exist.certificate": "भवतः समीपे विद्यमानं प्रमाणपत्रं अस्ति यस्य डोमेन् अथवा प्रमाणपत्रनाम भवता अनुरोधितं तथा च समाप्तेः समीपे नास्ति ।",
+	"certificate.dns01chanllenge.error.exist.timeout": "प्रतीक्षां कर्तुं समयः समाप्तः। कृपया पुनः प्रयासं कुर्वन्तु",
+	"certificate.dns01chanllenge.fail.above.message": "वयं निम्नलिखित DNS अभिलेखान् प्रमाणीकर्तुं असमर्थाः स्मः:",
+	"certificate.dns01chanllenge.fail.below.message": "भवता DNS अभिलेखाः निर्मिताः इति सुनिश्चितं कुर्वन्तु । यदि भवता कृतम् अस्ति तर्हि DNS परिवर्तनस्य प्रसारार्थं अधिककालं प्रतीक्षितुम् आवश्यकं भवेत् ।",
+	"certificate.dns01chanllenge.fail.info.modal.title": "केषाञ्चन डोमेनानां प्रमाणीकरणं कर्तुं असफलम् अभवत्",
+	"certificate.dns01chanllenge.modal.title.OK": "अस्तु",
+	"certificate.dns01chanllenge.modal.title.cancel": "निरसयतु",
+	"certificate.dns01chanllenge.modal.title.generate": "उद्- पद्",
+	"certificate.dns01chanllenge.modal.title.hostname": "होस्टनाम",
+	"certificate.dns01chanllenge.modal.title.txt.value": "TXT अभिलेखस्य मूल्यम्",
+	"certificate.dns01chanllenge.radio.title": "DNS-01 चुनौती",
+	"certificate.dns01chanllenge.requesting": "अनुरोधं कुर्वन्",
+	"certificate.dns01chanllenge.wait.above.message": "प्रमाणपत्रस्य होस्ट्नामानां स्वामित्वस्वामित्वं सत्यापयितुं कृपया निम्नलिखित DNS TXT अभिलेखान् योजयन्तु:",
+	"certificate.dns01chanllenge.wait.below.message": "एकदा भवता कृतं चेत्, निरन्तरं कर्तुं Generate नुदन्तु । ध्यानं कुर्वन्तु यत् नूतनानां DNS अभिलेखानां प्रसारणाय कतिपयानि निमेषाणि यावत् समयः भवितुं शक्नोति अतः वयं तान् पुष्टयितुं शक्नुमः ।",
+	"certificate.dns01chanllenge.wait.info.modal.title": "किं भवान् नूतनं प्रमाणपत्रं निर्मातुम् इच्छति ?",
```

Comment:
Jungho, the above language is Sanksrit and not Arabic.

ar_SA contains Arabic text. Can you please check if the BabelEdit is generating appropriate text for ar_SA?

This is not a blocker. But good to get it fixed.

Reply:
@chiraags 
BabelEdit selected "sa-In (Sanskrit - India)" for ar_SA.json automatically. Which one should we select?
And I updated BabelEdit to the new version recently. Is this probably the reason why it occurred?

Reply:
I haven't updated to the new version. Do you see an option to assign Arabic language?
SA stands for Saudi Arabia

Reply:
I fixed some text and changed the language for ar_SA.
Thanks